viewer_clients/web-client/scripts/codecheckerviewer/FileViewBC.js

```diff
@@ -206,7 +210,7 @@ return declare(BorderContainer, {
       editorHeader.suppressButton.setDisabled(true);
     }
 
-    CC_SERVICE.getSourceFileData(fileId, true, function(sourceFileData) {
+    CC_SERVICE.getSourceFileData(that.viewedFileId, true, function(sourceFileData) {
```

Comment:
Again: A thrift call may fail. On error it should log an error message to the JavaScript console.